### PR TITLE
Feat/stripe payments

### LINKED ISSUE #9: Guest should be able to do a one time donation
**What:** Guest should be able to do a one-time donation

**Why:** To allow people to donate anonymously

**How:**

- Save all the data we can from a user, this should include:
  - All required data by Stripe, including email address
  - User IP address
- Guest should be able to donate without creating an account

### LINKED ISSUE #13: User should be able to subscribe
**What:** User should be able to subscribe

User will only have 1 active subscription.

**How:**

- User should be able to subscribe with a given plan.

### LINKED ISSUE #25: Admin should be able to see the donation history
**What:** Admin should be able to see the donation history

**Why:** Admin can list and manage individual donation

**How:** 

- Admin should see a list with all donations
- Admin should be able to see the detail of each donation
- We won't tackle filtering or any other back-office features in this ticket

### LINKED ISSUE #26: Admin should be able to see the subscriptions history
**What:** Admin should be able to see the subscriptions history

**Why:** Admin can list and manage individual subscriptions

**How:** 

- Admin should see a list with all subscriptions
- Admin should be able to see the detail of each subscription
  - Admin should be able to see the donations related to a subscription
- We won't tackle filtering or any other back-office features in this ticket

### PR DESCRIPTION
**Pull request by castrolem**
**What:**
 Guest should be able to do a one-time donation.

**Why:**
To allow people to donate anonymously.
Closes #9, closes #13, closes 14, closes #25 and closes #26

**How:**
Providing a path to create one time donations from the homepage using Stripe.

### CODE REVIEW

File: db/migrate/20190822095006_create_donations.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+class CreateDonations < ActiveRecord::Migration[6.0]
+  def change
+    create_table :donations, id: :uuid do |t|
+      t.money :amount
+      t.string :card_id
+      t.string :customer_stripe_id
+      t.string :donation_type
+      t.string :customer_ip
+
```
Comment (orlando) [inline db/migrate/20190822095006_create_donations.rb:11]: @castrolem let's add a status field to this model, we can use `Enum` for this https://api.rubyonrails.org/v5.2.3/classes/ActiveRecord/Enum.html. The idea is to store every charge we do and know if these were successful or not.

File: db/schema.rb [hunk 2]
```diff
@@ -10,12 +10,34 @@
...
   enable_extension "plpgsql"
 
+  create_table "cards", id: :uuid, default: -> { "gen_random_uuid()" }, force: :cascade do |t|
+    t.uuid "user_id"
+    t.string "brand"
+    t.integer "exp_month"
+    t.integer "exp_year"
+    t.string "last_digits"
+    t.datetime "created_at", precision: 6, null: false
+    t.datetime "updated_at", precision: 6, null: false
```
Comment (orlando) [inline db/schema.rb:26]: @castrolem does stripe provide a card id? If that's the case let's store it here

**Review by orlando — DISMISSED** [review]

**Comment by castrolem** [comment]
* [x] Add card id from successful payment into `stripe_card_id` at `card`

**Review by orlando — APPROVED** [review]
LGTM

**Pull request merged**